docs/index.html

```diff
@@ -87,6 +87,7 @@ <h2 class="title">A 1x Engineer...</h2>
           <li>Is not defined by the computer they're using.</li>
           <li>May decorate their laptop and workspace in any way they like, and is respectful of others' decor (or lack there of), too.</li>
           <li>Isn't defined by myopic Tweetstorms by clueless VCs.</li>
+          <li>Stands up for others who are being critised unfairly</li>
```

Comment:
```suggestion
          <li>Stands up for others who are being criticized unfairly.</li>
```

Reply:
I think depends on which standard. British or American

Reply:
Not everyone that speaks English is American 😄. I will update the punctuation though.